### ISSUE TITLE
Can't resize services widget in dashboard

### ISSUE THREAD

**Issue by donhwy** [openmediavault/openmediavault#188]
I can't resize the services width. I can resize the system info width. For some reason that affects the width of services.
![services](https://user-images.githubusercontent.com/31188495/47398502-89133580-d6f1-11e8-94c6-898cb013c70b.png)

**Comment by donhwy** [comment]
Not sure how it ended up side by side like that. I moved services to see what would happen and now I can't get it back like that. To bad I liked it that way. I may have done that in version 3. I forget if this is an upgrade or not. 

One other thing is you can add more than one service or other window. 

Thanks

**Comment by votdev** [comment]
> Not sure how it ended up side by side like that. I moved services to see what would happen and now I can't get it back like that. To bad I liked it that way. I may have done that in version 3.

The code base is the same, so it should work like before. BUT, the ExtJS dashboard is really bad. If i could, i would replace or improve it asap.

**Comment by Ansem93** [comment]
So we can hope for a new dashboard??

**Comment by votdev** [comment]
> So we can hope for a new dashboard??

No. I've checked the sources of the latest ExtJS version and there are no relevant changes done to the code which will fix that issue.

**Comment by mi-hol** [comment]
Can this be closed?

**Comment by votdev** [comment]
> Can this be closed?

I have configured the stale bot to do NOT mark and close `known issue` labeled issues because i think these issues should be visible to users that try to find something that is related to their issue.

Please see https://github.com/openmediavault/openmediavault/commit/d69a93863e46812f1631ae4ced4d2e411a2de191

**Comment by mi-hol** [comment]
Currently its only labeled to affect 4.x which is EOL
Assuming it doesn't affect 5.x, why would there be an interest to keep it as open?

**Comment by votdev** [comment]
It affects all ExtJS based OMV versions, this includes 5.x. But the issue will also come up in OMV6 with the new UI where widgets will also have fixed widths and heights.

**Comment by mi-hol** [comment]
Seems to contradict the desire to use mobile devices as primary client for OMV

**Comment by ryecoaaron** [comment]
Personally, I don't think mobile devices should be the primary client.

**Comment by votdev** [comment]
> Seems to contradict the desire to use mobile devices as primary client for OMV

Desktop is the main target, but OMV6 will have a more responsive UI than <= OMV5 thanks to Angular.

**Comment by ThRPctmylSgltwH** [comment]
The containers widget does not display the uptime when viewing in certain configurations. All browsers are affected. I have tested with Edge, Chrome and Firefox. It does show on my phone using Chrome. I had a container that restarted every minute and missed this since I could not see the uptime on the dashboard. If screen is made very narrow, the third column with uptime is displayed. Can you rearrange the uptime column to column 1 and provide a slider to go from left to right?

one computer screen in several browsers - if screen is made very narrow, the third column with uptime is displayed.
![Screenshot 2022-08-30 081641](https://user-images.githubusercontent.com/86919018/187364829-e8a87a9d-17bd-42ec-9672-5d97d9f62f3a.jpg)

on phone
![Screenshot_20220830-082018_Chrome](https://user-images.githubusercontent.com/86919018/187365167-b2c4e468-e47b-4c8f-8659-79907334d018.png)

**Comment by ryecoaaron** [comment]
I will look at this when I get back from. Vmware Explore in a few days but it really should be an issue here - https://github.com/OpenMediaVault-Plugin-Developers/openmediavault-omvextrasorg/issues
